tap_mysql/__init__.py

```diff
@@ -136,6 +135,55 @@ def make_state_message(self):
         return singer.StateMessage(value=result)
 
 
+@attr.s
+class StreamMeta(object):
+    replication_key = attr.ib(default=None)
+    key_properties = attr.ib(default=None)
+    schema = attr.ib(default=None)
```

Comment:
this could be confusing once `StreamMeta` is used for a postgresql tap, which will have `database`, `table`, and an attribute for the postgres table schema.